utils_cv/detection/plot.py

```diff
@@ -37,30 +39,32 @@ def __init__(
 
 
 def plot_boxes(
-    im: JpegImageFile,
-    boxes: List[List[int]],
-    categories: List[str],
+    im: PIL.Image.Image,
+    anno_bboxes: List[AnnotationBbox],
```

Comment:
If this function also works for detected bounding boxes then I would rename "anno_bboxes" maybe into simply "bboxes". Same comment also applies to other places in this file.